sbt-scalajs-bundler/src/main/scala/scalajsbundler/sbtplugin/ScalaJSBundlerPlugin.scala

```diff
@@ -187,32 +202,8 @@ object ScalaJSBundlerPlugin extends AutoPlugin {
       *
       * @group tasks
       */
-    val webpack: TaskKey[Seq[File]] =
-      taskKey[Seq[File]]("Bundle the output of a Scala.js stage using webpack")
-
-    /**
-      * Bundles the output of a Scala.js stage using the reload workflow.
-      *
-      * This is equivalent to running fastOptJS::webpack with reloadWorkflow := true.
-      * This task must be scoped by the Scala.js fastOptJS stage.
-      *
-      * For instance, to bundle the output of `fastOptJS`, run the following task from the sbt shell:
-      *
-      * {{{
-      *   fastOptJS::webpackReload
-      * }}}
-      *
-      * To use a custom webpack configuration file use:
-      *
-      * {{{
-      *   webpackConfigFile in webpackReload := Some(baseDirectory.value / "webpack-reload.config.js"),
-      * }}}
-      *
-      * @group tasks
-      */
-    val webpackReload: TaskKey[Seq[File]] =
-      taskKey[Seq[File]]("Bundle the output of a Scala.js stage by appending the generated javascript to the pre-bundled dependencies")
-
+    val webpack: TaskKey[Seq[Attributed[File]]] =
+      taskKey[Seq[Attributed[File]]]("Bundle the output of a Scala.js stage using webpack")
```

Comment:
Could you add links to the possible types of attribute keys?

Reply:
Not sure what you're thinking here. As a scaladoc comment? The string in () is ui sugar for the REPL, right, so it doesn't get scaladoc processed, does it?

Reply:
No it’s not scaladoc processed :) You should add this documentation to the scaladoc of the `webpack` task. (my comment was on this line because it is where we see the type `TaskKey[Seq[Attributed[File]]]`)

Reply:
Ok, gotcha. Pushed some more stuff, let me know what you think.